cri/v1alpha2/cri.go

```diff
@@ -964,11 +1002,13 @@ func (c *CriManager) ContainerStatus(ctx context.Context, r *runtime.ContainerSt
 	}
 
 	podSandboxID := container.Config.Labels[sandboxIDLabelKey]
-	res, err := c.SandboxStore.Get(podSandboxID)
+
+	// get the sandbox's meta data.
```

Comment:
I found that some comments' first alphabet is upper, while others are lower. Should we use the upper style in comment?

Reply:
In general, we try to use lower case for comments in code snippets.